code/src/App.js

```diff
@@ -1,12 +1,32 @@
 import React from 'react'
 import data from './data.json'
+import { Header } from 'components/Header.js'
```

Comment:
It's completely fine to use `.js` in imports, but generally, best practice is to exclude it. So just `components/Header` :)